src/App.js

```diff
@@ -10,9 +10,11 @@ function App() {
       <Switch>
         <Route exact path="/">
           <h1>Welcome to Books-a-Million</h1>
```

Comment:
generally it's a lot cleaner to have just a component inside a ```<Route>``` instead of a bunch of HTML